Week1/homework/world-queries.txt

```diff
@@ -37,4 +37,181 @@ SELECT city.Name FROM city JOIN country ON city.CountryCode = country.Code WHERE
 | Heerlen           |
 | Alkmaar           |
 +-------------------+
-28 rows in set (0.00 sec)
\ No newline at end of file
+28 rows in set (0.00 sec)
```

Comment:
Try not to add so much to each PR, since it's not actually relevant. You've included all the previous homework stuff in the week3 PR, which just makes it more difficult for your reviewer to find what's relevant.